vsphere/resource_vsphere_virtual_machine_test.go

```diff
@@ -7178,11 +7178,17 @@ resource "vsphere_virtual_machine" "vm" {
   datacenter_id              = data.vsphere_datacenter.rootdc1.id
   host_system_id             = data.vsphere_host.roothost1.id
   wait_for_guest_net_timeout = 0
-  wait_for_guest_ip_timeout  = 1
+  wait_for_guest_ip_timeout  = 0
```

Comment:
Why are we changing this? I'm concerned that it may surprise users depending on existing behaviour.

Reply:
Thats just to help speed up the testing. Not all ovf/ova sources have vmware tools installed, so (along with speeding things up) I didn't want to get acc test failures based on that.

Reply:
You are right, I shouldn't be reviewing code this early :)

Reply:
Ha! And I shouldn't be submitting a PR this late!